lib/engine.py

```diff
@@ -290,19 +291,19 @@ def create_db(self):
         engine.db"""
         db_name = self.database_name()
         if db_name:
-            print "Creating database " + db_name + "..."
+            print("Creating database " + db_name + "...")
             # Create the database
             create_stmt = self.create_db_statement()
             if self.debug:
-                print create_stmt
+                print(create_stmt)
             try:
                 self.execute(create_stmt)
             except Exception as e:
                 try:
                     self.connection.rollback()
                 except:
                     pass
-                print "Couldn't create database (%s). Trying to continue anyway." % e
+                print("Couldn't create database (%s). Trying to continue anyway." % e)
```

Comment:
There's a lot of these old-style format in the code, and changing that will be a lot of manual changes . Since both are supported in python 3, I think we could create a issue and delay it until important changes are done?